addons/doris/templates/backuppolicytemplate.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: BackupPolicyTemplate
+metadata:
+  name: {{ include "doris.name" . }}-backup-policy-template
+  labels:
+    clusterdefinition.kubeblocks.io/name: doris # 通过 label 指定作用域，必须填
```

Comment:
please use English

Reply:
thanks, I have checked the Doris-related parts of the code and modified all of them which is not using English